roles/gNb/tasks/stop.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: delete {{ srsran.docker.container.gnb_image }} container
+  community.docker.docker_container:
+    name: "rfsim5g-srsran-gnb"
+    state: absent
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+- name: remove /tmp/gnb_zmq.conf
+  file:
+    path: "/tmp/gnb_zmq.conf"
+    state: absent
+  when: inventory_hostname in groups['srsran_nodes']
+  become: true
+
+  #TODO: Enable once we started using images from docker hub
+  #- name: remove {{ srsran.docker.container.gnb_image }} image
+  # community.docker.docker_image:
+  #  name: "{{ srsran.docker.container.gnb_image }}"
+  #  state: absent
+  #  force_absent: true
+  #when: inventory_hostname in groups['srsran_nodes']
+  #become: true
```

Comment:
We already have the images in DockerHub (`https://hub.docker.com/u/aetherproject`), why keeping this `TODO` here?

Reply:
when this review raised we dont have images in DockerHub so i added this TODO. Now i enabled it.